tensorboard/webapp/app_routing/store_only_utils.ts

```diff
@@ -30,13 +31,16 @@ import {CompareRouteParams, Route} from './types';
  */
 export function getCompareExperimentIdAliasSpec(
   routeParams: CompareRouteParams
-): Map<string, string> {
-  const idToDisplayName = new Map<string, string>();
+): Map<string, ExperimentAlias> {
+  const idToDisplayName = new Map<string, ExperimentAlias>();
   const nameAndIds = parseCompareExperimentStr(routeParams.experimentIds);
+  let aliasNumber = 0;
   for (const {id, name} of nameAndIds) {
-    if (name) {
-      idToDisplayName.set(id, name);
+    aliasNumber++;
```

Comment:
Is it contract of the utility that the aliasNumber starts from 1 instead of 0? That looks a lot like presentational logic and not data logic which is weird to find in the data util. 

Anyways, if it should start from 1, please document it on jsdoc since that contract is not documentable with types alone.

Reply:
Documented.